src/main/java/uk/gov/hmcts/reform/cwrdapi/domain/RoleType.java

```diff
@@ -48,5 +48,9 @@
     @OneToMany(targetEntity = CaseWorkerIdamRoleAssociation.class, mappedBy = "roleType")
     private List<CaseWorkerIdamRoleAssociation> caseWorkerIdamRoleAssociations = new ArrayList<>();
 
+    public RoleType(String description) {
```

Comment:
Can't we use the builder pattern to create objects instead of adding a constructor?

Reply:
I reproduced in this class so I will post it here

warning: @Builder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
    private List<CaseWorkerIdamRoleAssociation> caseWorkerIdamRoleAssociations = new ArrayList<>();
    
  same error for this field:
  private List<CaseWorkerRole> caseWorkerRoles = new ArrayList<>();

Reply:
I tried adding Builder.Default, didn't work.

Tried making the fields final and this led to more errors.

Reply:
yeah, seeing the same issue :) thanks for your explanation